### DISCUSSION TITLE
Docker 部署 出现大量 [GET] "..." <no response> This operation was aborted

### DISCUSSION THREAD

**Discussion by Jiematcha** [DIYgod/RSSHub#15059] [Q&A] [answered]
4/1 更新后高频率出现 Feeds 503 问题
Error Message 都是 [GET] "..." <no response> This operation was aborted 格式
各种 Feed 都可能出现

/weibo/user/3511690595
[GET] "https://m.weibo.cn/statuses/show?id=NEvsispVL": <no response> This operation was aborted: target website might be blocking our access, you can host your own RSSHub instance for a better usability.

/bilibili/user/dynamic/946974
[GET] "https://api.bilibili.com/x/polymer/web-dynamic/v1/feed/space?host_mid=946974&platform=web&features=itemOpusStyle%2ClistOnlyfans%2CopusBigCover%2ConlyfansVote": <no response> This operation was aborted: target website might be blocking our access, you can host your own RSSHub instance for a better usability.

/zaobao/realtime/world
[GET] "https://www.zaobao.com/realtime/world": <no response> This operation was aborted: target website might be blocking our access, you can host your own RSSHub instance for a better usability.

**Comment by DIYgod** [comment, marked as answer]
This is caused by `requestTimeout`, the default maximum wait time for each request is 30 seconds. If a longer waiting time is needed due to very poor network conditions, please set the `REQUEST_TIMEOUT` environment variable.

**Reply by Jiematcha** [reply]
我发现是代理问题，之前使用的socks5h未出现问题，更新后socks5 sock5h，我的代理服务器都收不到，没有记录，换成http代理后正常了
